### PR TITLE
Feature/autoproj

### PR DESCRIPTION
**Pull request by aarodrigues**
- #### Use the following settings in the overrides file for docker testing
 ```yaml
overrides:
  - pkg_set:github:/Brazilian-Institute-of-Robotics/bir.digisub-package_set.git:
    branch: master
  - uw-calibration-pinax:
    github: Brazilian-Institute-of-Robotics/uw-calibration-pinax
    branch: feature/autoproj
 ```
- #### What was added/changed in this update
   - Configure project to run using ros metapackage structure
- #### Depens on:
   N/A

### CODE REVIEW

File: jir_refractive_image_geometry/package.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  <!-- The export tag contains other, unspecified, tags -->
+  <export>
+    <!-- You can specify that this package is a metapackage here: -->
+    <!-- <metapackage/> -->
+
+    <!-- Other tools can request additional information be placed here -->
+
+  </export>
+</package>
```
Comment (ronaldoevangelista) [inline jir_refractive_image_geometry/package.xml]: no new line at end of file.

is this a third party package?

File: uw-calibration-pinax/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(uw-calibration-pinax)
+find_package(catkin REQUIRED)
+
+catkin_metapackage()
```
Comment (ronaldoevangelista) [inline uw-calibration-pinax/CMakeLists.txt]: no newline at end of file

Comment (aarodrigues) [reply uw-calibration-pinax/CMakeLists.txt]: Done

File: jir_refractive_image_geometry/package.xml (hunk 1 above)
Comment (aarodrigues) [reply jir_refractive_image_geometry/package.xml]: Yes

**Review by NestorDP — APPROVED** [review]

**Review by ronaldoevangelista — APPROVED** [review]
feel free to merge

**Pull request merged**